### ISSUE TITLE
gcc-9 does not work with ALPAKA_ACC_CPU_BT_OMP4_ENABLE in CI

### ISSUE THREAD

**Issue by BenjaminW3** [alpaka-group/alpaka#817]
The current gcc-9 target uses `ALPAKA_ACC_CPU_BT_OMP4_ENABLE=OFF` in CI due to:
```
/usr/bin/g++  -g   CMakeFiles/axpy.dir/src/axpy.cpp.o  -o axpy ../../common/libcommon.a /home/travis/boost_libs/lib/libboost_fiber-mt-d.a /home/travis/boost_libs/lib/libboost_context-mt-d.a /home/travis/boost_libs/lib/libboost_thread-mt-d.a -lpthread /home/travis/boost_libs/lib/libboost_chrono-mt-d.a /home/travis/boost_libs/lib/libboost_system-mt-d.a /home/travis/boost_libs/lib/libboost_date_time-mt-d.a /home/travis/boost_libs/lib/libboost_atomic-mt-d.a /usr/lib/x86_64-linux-gnu/libtbbmalloc.so /usr/lib/x86_64-linux-gnu/libtbb.so -lpthread -lrt -fopenmp ../../libCatchMain.a 
lto-wrapper: fatal error: could not find accel/nvptx-none/mkoffload in /usr/lib/gcc/x86_64-linux-gnu/9/:/usr/lib/gcc/x86_64-linux-gnu/9/:/usr/lib/gcc/x86_64-linux-gnu/:/usr/lib/gcc/x86_64-linux-gnu/9/:/usr/lib/gcc/x86_64-linux-gnu/ (consider using '-B')
```

**Comment by jkelling** [comment]
This is actually not a problem with the alpaka OpenMP4 backend, but with the CI setup: Either the gcc was not compiled with the nvptx backend enabled, or the environment is not configured correctly to find the backend.

**Comment by BenjaminW3** [comment]
Yes, it is not at all a problem of alpaka. It is a problem in CI. We are uing a stock gcc-9 without any modifications. The same setup works with older gcc versions. Therefore, it is probably a miconfiguration of the gcc that is distributed.

**Comment by j-stephan** [comment]
I'm not sure if we should fix this (i.e., reenable the tests for gcc-9). @jkelling, what do you think?

**Comment by psychocoderHPC** [comment]
I will close this issue because we do not have anymore the OMP4 backend. We have only OMP2 and OMP5. gcc 9 is definitive not supporting OMP5. Not sure which gcc is fully supporting OMP5, GCC is in some newer versions claiming that they support OMP5 but only implemented OMP 4.5.

Nevertheless @jkelling could you add a note here which version of GCC is supporting OMP5.
